dGame/dUtilities/VanityUtilities.cpp

```diff
@@ -22,9 +22,18 @@
 
 #include <fstream>
 
-std::vector<VanityObject> VanityUtilities::m_Objects = {};
-std::set<std::string> VanityUtilities::m_LoadedFiles = {};
 
+namespace {
+	std::vector<VanityObject> Objects;
```

Comment:
Lgtm, incidentally. Would like to see the former members here uncapitalized (i.e. objects instead of Objects) but it's no big deal and I'll approve once the merge conflict is sorted

Reply:
I think its good to keep file scope variables with this syntax so you know what you may be referring to at a glance (file scope container vs local scope)

Reply:
Want me to change it back then?

Reply:
Not worth blocking further.  Though I believe pascal case for file scope is better.